<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Browse_for_InputFile">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse Input File</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_VideoQuality">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilePathField">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_SelectFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select a file:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DownloadOptions">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Download options</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_VideoQuality">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Video Quality</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Browse_for_DownloadFolder">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for Download location</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_SelectFile_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Download Folder:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SelectFile_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Download location:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DownloadVideos">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Download Videos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
